### DISCUSSION TITLE
[CLOSED] Which model explainability tool to integrate with Q1 2022

### DISCUSSION THREAD

**Discussion by htahir1** [zenml-io/zenml#235] [Show and tell]
# [VOTE] Which model explainability tool to integrate with Q1 2022

Which of the following commonly-used model explainability tools would you like us to support?

[Alibi](https://github.com/SeldonIO/alibi) > Vote with 👍
[Dalex](https://dalex.drwhy.ai) > Vote with 😀
[ELI5](https://eli5.readthedocs.io/en/latest/index.html) > Vote with 🎉
[InterpretML](https://interpret.ml) > Vote with ❤️
[Lime](https://github.com/marcotcr/lime) > Vote with 🚀
[What-If Tool](https://pair-code.github.io/what-if-tool/) > Vote with 👀

If you have another suggestion, leave a comment!

**Comment by mbertani** [comment]
I wouldn't go with LIME. The technique is nice from a theoretical perspective, and it was one of the early methods that started the XAI movement. It has unfortunately a lot of issues (see f.ex. https://github.com/dylan-slack/Fooling-LIME-SHAP) that make it difficult to trust the results of this algorithm.

**Reply by htahir1** [reply]
Interesting thought!
